test/functional/repository/remove_content_test.rb

```diff
@@ -104,5 +104,21 @@ module HammerCLIKatello
                    --ids 20,21,22))
       end
     end
+
+    it 'does not resolves repository id without product details' do
```

Comment:
This test already exists: https://github.com/Katello/hammer-cli-katello/blob/master/test/functional/repository/remove_content_test.rb#L18